### DISCUSSION TITLE
How to save model by code?

### DISCUSSION THREAD

**Discussion by afunrwth** [Dlubal-Software/RSTAB_Python_Client#7] [Q&A] [answered]
How do I save a new created model by code?  "closeModel('..', True) does not work for me (?).

**Comment by heetrojivadiya** [comment, marked as answer]
Hello, you can save model with code by using '[saveFile](https://github.com/Dlubal-Software/RSTAB_Python_Client/blob/main/RSTAB/initModel.py#LL287C1-L287C1)' by importing 'from RSTAB.initModel import saveFile' or you can use 'Model.clientModel.service.save(file_path)'. I hope this will work for you.
